com.sap.cloud.lm.sl.cf.database/src/test/java/com/sap/cloud/lm/sl/cf/database/migration/generator/DatabaseTableInsertQueryGeneratorTest.java

```diff
@@ -0,0 +1,64 @@
+package com.sap.cloud.lm.sl.cf.database.migration.generator;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.DatabaseTableColumnMetadata;
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.DatabaseTableMetadata;
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.ImmutableDatabaseTableColumnMetadata;
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.ImmutableDatabaseTableMetadata;
+
+class DatabaseTableInsertQueryGeneratorTest {
+    private static final String TEST_TABLE_NAME = "testTableName";
+    private static final String TEST_COLUMN_NAME = "testColumnName";
+    private static final String TEST_COLUMN_TYPE = "testColumnType";
+
+    private DatabaseTableInsertQueryGenerator databaseTableInsertQueryGenerator;
```

Comment:
Just initialize the field here, so that you don't have to initialize it in every test.